README.md

```diff
@@ -52,7 +52,7 @@ jobs:
           prerelease: false
       - name: Upload Release Asset
         id: upload-release-asset 
-        uses: actions/upload-release-asset@v1.0.1
+        uses: actions/upload-release-asset@v1
```

Comment:
For what it's worth, this change didn't work for me. I get:
```
Download action repository 'actions/upload-release-asset@v1'
##[warning]Failed to download action 'https://api.github.com/repos/actions/upload-release-asset/tarball/v1'. Error Response status code does not indicate success: 404 (Not Found).
```

Reply:
This should be resolved now